packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -719,11 +719,15 @@ function mountIndeterminateComponent(
     Component !== null &&
     typeof Component.then === 'function'
   ) {
+    // We can't start a User Timing measurement with correct label yet.
+    // Cancel and resume right after we know the tag.
+    cancelWorkTimer(workInProgress);
```

Comment:
does this make two segments in the flame chart?

Reply:
No, it just discards the in-progress segment.